tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -21,11 +21,10 @@ module.exports = {
      * Checks if the file-row of the desired file-name with the username consists of the desired status of accepted,
      * declined or pending
      */
-    assertDesiredStatusIsPresent: function (filename, status, user) {
+    assertDesiredStatusIsPresent: function (filename, status, user = undefined) {
       let requiredXpath = this.api.page.FilesPageElement.filesList().getFileRowSelectorByFileName(filename) +
-        util.format(this.elements.assertStatusFileRow.selector, status)
-      requiredXpath = user === undefined ? requiredXpath : requiredXpath +
-                      util.format(this.elements.getSharedFromUserName.selector, user)
+                          util.format(this.elements.assertStatusFileRow.selector, status)
+      requiredXpath = user !== undefined ? requiredXpath + util.format(this.elements.getSharedFromUserName.selector, user) : requiredXpath
```

Comment:
why do these lines need to be changed?
IMO the logic has been sswapped around, but it ends up the same behavior.